docs/developers/platform.md

```diff
@@ -97,3 +49,96 @@ subgraph CLI
   cli[CLI]
 end
 </div>
+
+* [CLI][cli] - Command Line Interface for publishing [Data Packages](#data-package)
+* [Front-end Web Application][web-app] - Core part of platform - API, Login & Sign-Up and Browse & Search (page not yet implemented)
+* [Views and Renderer][views] - JS Library responsible for visualization and views on platform
+* [BitStore](#bitstore) - Place were data is stored
+* [MetaStore](#metastore) - Place for storing and managing package meta-data, users, permissions etc
```

Comment:
I think we don't need to have bullet points for MetaStore, BitStore and Users & Permissions, because it comes right after it.